### ISSUE TITLE
Add constraints to field

### ISSUE THREAD

**Issue by quangIO** [requery/requery#813]
How can I have something like this?
CREATE TABLE foo (
  someint INT NOT NULL CHECK (someint >= 0 AND someint <= 10)
)

**Comment by hc-codersatlas** [comment]
Would be good to integrate with Hibernate Validator (and the validation api in general) since it's often used in REST APIs too.

**Comment by antcasq** [comment]
Yep, I was hopping I could use Bean Validation 2.0 (JSR 380) https://beanvalidation.org/ with requery to specify constraints on the database columns.
